src/components/pages/HomePage/index.tsx

```diff
@@ -39,34 +43,165 @@ export default function HomePage() {
 
   return (
     <main>
-      <Typography
-        variant='h3'
-        sx={{ color: (theme) => theme.palette.common.white }}
-      >
-        Welcome to Frempco!
-      </Typography>
-
-      <Grid container spacing={2}>
-        <Grid item xs={12} md={4}>
-          <Box sx={{ m: 5 }}>
-            <Typography variant='h3' sx={{ color: 'white', mb: 1 }}>
-              Students:
+      <Box color={(theme) => theme.palette.common.white}>
+        <Grid container textAlign='center'>
+          <Grid item md={12} p={2}>
+            <Typography variant='h3' textAlign='left' mb={6}>
+              Frempco
             </Typography>
-            <StudentsButton visitStudentsPage={visitStudentsPage} />
+          </Grid>
 
-            <Typography variant='h3' sx={{ color: 'white', mb: 1, mt: 8 }}>
-              Teachers:
-            </Typography>
-            <TeachersButton visitTeachersPage={visitTeachersPage} />
-          </Box>
+          <Grid item sm={12} md={7} p={2}>
+            <Box maxWidth='700px' mx='auto'>
+              <Typography variant='h4' textAlign='center' mb={3}>
+                Don&apos;t just tell your students. Show them.
+              </Typography>
+              <Typography fontSize='28px'>
```

Comment:
24px looks better on mobile while still looking fine on desktop
```suggestion
              <Typography fontSize='24px'>
```